Modules/itertoolsmodule.c

```diff
@@ -4473,7 +4473,7 @@ zip_longest_next(ziplongestobject *lz)
         Py_INCREF(result);
         for (i=0 ; i < tuplesize ; i++) {
             it = PyTuple_GET_ITEM(lz->ittuple, i);
-            if (it == NULL) {
+            if (it == Py_None) {
```

Comment:
```suggestion
            if (Py_IsNone(it)) {
```

Reply:
I really don't like these sort of macros. They offer no safety or portability improvements, in fact they will probably impede any future work on tagging or partitioning references.